src/views/Bond/components/BondModal/BondModal.tsx

```diff
@@ -158,6 +158,6 @@ const BondModal: React.VFC<{ bond: Bond }> = ({ bond }) => {
 const TokenPrice: React.VFC<{ token: Token; isInverseBond?: boolean }> = ({ token, isInverseBond }) => {
   const { data: priceToken = new DecimalBigNumber("0") } = useTokenPrice({ token, networkId: NetworkId.MAINNET });
   const { data: ohmPrice = 0 } = useOhmPrice();
-  const price = isInverseBond ? priceToken.mul(new DecimalBigNumber(ohmPrice.toString())) : priceToken;
+  const price = isInverseBond ? new DecimalBigNumber(ohmPrice.toString()) : priceToken;
```

Comment:
Is this related to the bug that was reported? What was the previous rationale for having `priceToken.mul`, and how is it different now?

Reply:
1. yes
2. I did not go back to verify but I believe we just mistakenly implemented this multiplication both here & on `BondPrice` when we inverted the logic for InverseBonds (if you remember the original Inverse Bond UI treated Inverse Bonds like normal bonds where the user was "buying" DAI). Inverting all that logic was a little funky & I think we just made a mistake.

Reply:
@brightiron regarding what you're saying about this change potentially being an issue on ETH bonds (or something other than stables)...

Shouldn't `TokenPrice` always show the straight price of OHM on Inverse Bonds? Even if it's an Inverse Bond for ETH TokenPrice should still be the price of OHM. `BondPrice` is where the logic needs to take into account the `quoteTokenPerBaseToken` price that the bond contract gives us.

Reply:
when you say display the market price in ETH that would be something like `0.012 ETH`?

Reply:
> when you say display the market price in ETH that would be something like `0.012 ETH`?

yes, exactly. If i sold 1 OHM on open market i'd receive 0.01212 ETH.